src/profile/webid.ts

```diff
@@ -58,6 +58,7 @@ export function getAltProfileUrlAllFrom(
   const altProfileUrlAll = getThingAll(webIdProfile)
     .filter((thing) => getIriAll(thing, foaf.primaryTopic).length > 0)
     .map(asIri)
+    .concat(webIdThing ? getIriAll(webIdThing, rdfs.seeAlso) : [])
```

Comment:
Should we have some de-duplication logic here for profiles that use both `rdfs:seeAlso` and `foaf:isPrimaryTopicOf` to refer to another WebId?

Reply:
Sounds like a good idea, e.g. reducing into a Set.

Reply:
If I am not misunderstanding the comment, that is already done in L66.

Reply:
Ah - my bad!